functional/unit_action_test.go

```diff
@@ -69,47 +76,132 @@ func TestUnitSubmit(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	unitFile := "fixtures/units/hello.service"
+	inputUnitFiles := make([]string, 0)
+	inputUnitFiles = append(inputUnitFiles, path.Base(unitFile))
+
 	// submit a unit and assert it shows up
-	if _, _, err := cluster.Fleetctl(m, "submit", "fixtures/units/hello.service"); err != nil {
+	_, _, err = cluster.Fleetctl(m, "submit", unitFile)
+	if err != nil {
```

Comment:
Why did you change the syntax here? There is no reason to have the assignment outside the `if`.